src/org/parosproxy/paros/core/scanner/VariantAbstractQuery.java

```diff
@@ -163,7 +168,8 @@ private String setParameter(HttpMessage msg, NameValuePair originalPair, String
         StringBuilder sb = new StringBuilder();
         String encodedValue = (escaped) ? value : getEscapedValue(msg, value);
         NameValuePair pair;
-        boolean isAppended;
+        boolean isAppended = false;

```

Comment:
Strange, that's not used outside the loop (and inside the loop it's set before being used).